presto-docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -23,6 +23,39 @@ connection properties as appropriate for your setup:
     connection-user=root
     connection-password=secret
 
+
+Connection security
+------------------
+
+The JDBC driver, and therefore the connector, automatically use Transport Layer Security (TLS) encryption and certificate validation. This requires a suitable TLS certificate configured on your SQL Server database host.
+
+If you do not have the necessary configuration established, you can disable encryption in the connection string with the encrypt property:
+
+.. code-block:: none
+
+    connection-url=jdbc:sqlserver://<host>:<port>;databaseName=<databaseName>;encrypt=false;
+
+Further parameters like trustServerCertificate, hostNameInCertificate, trustStore, and trustStorePassword can be configured as below
+
+SSL Configuration Properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+================================================== ==================================================================== ===========
+parameters                                         Description                                                          Default
```

Comment:
```suggestion
Property Name                                      Description                                                          Default
```
Suggest column name change for consistency with other pages in [Connectors](http://prestodb.io/docs/current/connector.html). I tested this suggestion in a local build.